collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -83,35 +133,299 @@
+                           <explicit>Et quia non adest iustus qui corripiat in misericordia oleum peccatoris inpinguet caput eorum.</explicit>
+                        </msItem>
+                        <note>The main authors named in the rubrics are <persName key="person_100184667" role="aut">Gregory</persName> and <persName key="person_61539765" role="aut">Bede</persName>; others are <persName key="person_100227669" role="aut">Ambrose</persName>, <persName key="person_95155322" role="aut">Origen</persName>, <persName key="person_95147024" role="aut">Jerome</persName>, and <persName key="person_297475745" role="aut">John Chrysostom</persName>.</note>
                      </msItem>
                   </msContents>
                   <physDesc>
                      <objectDesc form="codex">
                         <supportDesc material="perg">
-                           <support>parchment</support>
+                           <support>Parchment; some original flaws with medieval repairs; many blank margins excised or sliced, with occasional loss of small amounts of text</support>
+                           <extent>113 leaves</extent>
                         </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="28 30">Ruled in plummet for two columns of 28–30 lines</layout>
+                        </layoutDesc>
                      </objectDesc>
+                     <handDesc>
+                        <handNote script="protogothic">Protogothic bookhands, by more than one scribe (e.g. there is a change in the second column of fol. 2r).</handNote>
+                     </handDesc>
+                     <decoDesc>
+                        <decoNote type="decInit">One fine pen-drawn zoomorphic initial in the form of a dragon sprouting foliage, reserved against a red ground (fol. 35r); other initials with lesser decoration in plain red, sometimes effusively embellished (e.g. fol. 45v, 78r).</decoNote>
+                     </decoDesc>
                   </physDesc>
                   <history>
                      <origin>
-                        <origDate calendar="Gregorian" notAfter="1200" notBefore="1100">12th century</origDate>
+                        <origDate calendar="Gregorian" notAfter="1200" notBefore="1170">12th century, last(?) quarter</origDate>
                         <origPlace>
                            <country key="place_7000084">German</country>
                         </origPlace>
                      </origin>
+                     <provenance>The book has common provenance, as yet unidentified, with MS. Laud Misc. 465: both have margins excised in an identical manner, with added replacement text (e.g. here fol. 49) and flex punctuation (e.g fols. 106–7).</provenance>
+                     <provenance>12th–15th-century marginalia (e.g. fols. 17v, 19r, 35r).</provenance>
                   </history>
```

Comment:
do these two provenance comments apply only to this section or to the whole volume? If the latter, they should be moved to line c. 64 above; it looks like the excisions at least characterize the whole volume.